torch/fx/experimental/validator.py

```diff
@@ -534,7 +534,9 @@ def add_assertion(self, e: Union[z3.BoolRef, sympy.Basic]) -> None:
             self._assertions.add(ref)
 
         def validate(self) -> None:
-            with dynamo_timed("TranslationValidator.validate"):
+            with dynamo_timed(
+                "TranslationValidator.validate", log_pt2_compile_event=False
```

Comment:
This is probably OK to emit a row for, it's O(number of compilations)

Reply:
Can do, it was just False before (because the previous default was False)